### PR TITLE
feat(payment): STRIPE-418 StripeUPE E2E tests added

### PR DESCRIPTION
**Pull request by bc-dronov**
## What?
Added E2E tests for Stripe-integration package

## Testing / Proof
All tests have been passed

@bigcommerce/team-checkout

### CODE REVIEW

**Review by PavlenkoM — APPROVED** [review]

**Review by PavlenkoM — APPROVED** [review]

File: packages/stripe-integration/e2e/support/stripeUPE.mock.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    window.Stripe = () => ({
+        elements: () => ({
+            create: stripeElement,
+            getElement: stripeElement,
+            update: stripeElement,
+            fetchUpdates: stripeElement,
+        }),
+    })
+})()
```
Comment (animesh1987) [inline packages/stripe-integration/e2e/support/stripeUPE.mock.js:24]: EOL

File: packages/test-framework/src/fixture/pagePreset/PaymentStepAsGuest.ts [hunk 2]
```diff
@@ -9,14 +9,15 @@ export class PaymentStepAsGuest implements CheckoutPagePreset {
...
+        private postalCode?: string,
     ) {}
     async apply(page: Page): Promise<void> {
         const api = new ApiRequestsSender(page, this.locale);
 
         await api.addPhysicalItemToCart();
         await api.setCurrency(this.currency);
         await api.completeCustomerStepAsGuest();
-        await api.completeSingleShippingAndSkipToPaymentStep(this.countryCode);
+        await api.completeSingleShippingAndSkipToPaymentStep(this.countryCode, this.postalCode);
```
Comment (animesh1987) [inline packages/test-framework/src/fixture/pagePreset/PaymentStepAsGuest.ts:20]: Why the change here?

File: packages/stripe-integration/e2e/support/stripeUPE.mock.js (hunk 1 above)
Comment (bc-dronov) [reply packages/stripe-integration/e2e/support/stripeUPE.mock.js]: Done

File: packages/test-framework/src/fixture/pagePreset/PaymentStepAsGuest.ts (hunk 2 above)
Comment (bc-dronov) [reply packages/test-framework/src/fixture/pagePreset/PaymentStepAsGuest.ts]: Removed. This code is no more needed because I mocked Stripe config.
